docs/guide/service/annotations.md

```diff
@@ -14,7 +14,8 @@
 | service.beta.kubernetes.io/aws-load-balancer-type                              | string     |                           |                        |
 | service.beta.kubernetes.io/aws-load-balancer-internal                          | boolean    | false                     |                        |
 | [service.beta.kubernetes.io/aws-load-balancer-proxy-protocol](#proxy-protocol-v2)                 | string     |        | Set to `"*"` to enable |
-| service.beta.kubernetes.io/aws-load-balancer-access-log-enabled                | boolean    | false                     |                        |
```

Comment:
keep the aws-load-balancer-access-log-enabled annotation as is, and add a row for aws-load-balancer-ip-address-type instead of aws-load-balancer-type

Reply:
Yeah sorry, it's been a long day 😆 

It should be correct now.